source/getting-started/z-wave.markdown

```diff
@@ -58,7 +58,7 @@ With this installation, your `config_path` needed below will resemble:
 
 If you followed along with setting up a virtual environment, your path will be:
 ```bash
-/srv/hass/python-openzwave/openzwave/config
+/srv/homeassistant/python-openzwave/openzwave/config
```

Comment:
This used to be correct if you followed the virtualenv guide but it was changed to this in 34d80da743a634fbc3175433f6cb2f49dc728f1e